classifiers/dns.go

```diff
@@ -0,0 +1,30 @@
+package classifiers
+
+import (
+	"github.com/google/gopacket/layers"
+	"github.com/mushorg/go-dpi"
+)
+
+type DnsClassifier struct{}
+
+func (_ DnsClassifier) HeuristicClassify(flow *godpi.Flow) bool {
+	if len(flow.Packets) == 0 {
+		return false
+	}
+	for _, packet := range flow.Packets {
+		if layer := (*packet).Layer(layers.LayerTypeUDP); layer != nil {
+			srcPort := layer.(*layers.UDP).SrcPort
+			dstPort := layer.(*layers.UDP).DstPort
+			if srcPort != 53 && dstPort != 53 {
+				return false
+			}
+		} else {
+			return false
+		}
+	}
+	return true
```

Comment:
If it passes the rest of the checks (all packets having the appropriate src or dst port) then I return true.